stackit/internal/services/git/instance/resource.go

```diff
@@ -307,12 +357,61 @@ func mapFields(resp *git.Instance, model *Model) error {
 		return fmt.Errorf("git instance id not present")
 	}
 
+	var aclList basetypes.ListValue
+	var diags diag.Diagnostics
+	if model.ACL.IsUnknown() || model.ACL.IsNull() {
+		aclList = types.ListNull(types.StringType)
+	} else {
+		aclList, diags = types.ListValueFrom(ctx, types.StringType, resp.Acl)
+		if diags.HasError() {
+			return fmt.Errorf("mapping ACL: %w", core.DiagsToError(diags))
+		}
+	}
+
+	if model.Created.IsUnknown() || model.Created.IsNull() {
+		model.Created = types.StringNull()
+	} else {
+		model.Created = types.StringValue(resp.Created.String())
+	}
+
 	// Build the ID by combining the project ID and instance id and assign the model's fields.
 	model.Id = utils.BuildInternalTerraformId(model.ProjectId.ValueString(), *resp.Id)
-	model.Url = types.StringPointerValue(resp.Url)
-	model.Name = types.StringPointerValue(resp.Name)
+	model.ACL = aclList
+	model.ConsumedDisk = types.StringPointerValue(resp.ConsumedDisk)
+	model.ConsumedObjectStorage = types.StringPointerValue(resp.ConsumedObjectStorage)
+	model.Flavor = types.StringPointerValue(resp.Flavor)
 	model.InstanceId = types.StringPointerValue(resp.Id)
+	model.Name = types.StringPointerValue(resp.Name)
+	model.Url = types.StringPointerValue(resp.Url)
 	model.Version = types.StringPointerValue(resp.Version)
 
 	return nil
 }
+
+// createPayloadFromModel creates the payload to create a git instance
+func createPayloadFromModel(ctx context.Context, model *Model) (git.CreateInstancePayload, diag.Diagnostics) {
```

Comment:
```suggestion
func toCreatePayload(ctx context.Context, model *Model) (git.CreateInstancePayload, diag.Diagnostics) {
```

It's not really a convention, but I think we can keep the naming the same as in the other resource implementations if it doesn't hurt you :sweat_smile: 

see https://github.com/search?q=repo%3Astackitcloud%2Fterraform-provider-stackit+toCreatePayload&type=code

Hint: Don't forget to rename the testcase ^^